### PR TITLE
feat: add theme v2 to tab navigation templates

### PR DESCRIPTION
**Pull request by MartoYankov**
TODO:

- [x] Add t-36 class to theme
- [x] The .ns-statusbar-transparent class doesn't work with Android Api19
- [x] Support dark theme switch
- [x] ActionBar title isn't centered on Android
- [x] ActionBar title isn't colored right on iOS
- [x] ListView items on NG aren't styled properly
- [ ] Add Tabs to home page and move master detail to browse page
- [ ] Remove unused resources
- [ ] Create new screen shots
- [ ] Update Readme

### CODE REVIEW

File: packages/template-tab-navigation-ng/src/app.android.scss [hunk 1]
```diff
@@ -1,11 +1,4 @@
-// Import app variables
-@import 'app-variables';
-
-// Import the theme’s main ruleset - both index and platform specific.
-@import '~nativescript-theme-core/scss/index';
-@import '~nativescript-theme-core/scss/platforms/index.android';
-
 // Import common styles
-@import 'app-common';
+@import "app-common";
```
Comment (bundyo) [inline packages/template-tab-navigation-ng/src/app.android.scss]: Remove these files and add an app.scss. :slightly_smiling_face:

File: packages/template-tab-navigation-ng/src/_app-common.scss [hunk 2]
```diff
@@ -12,8 +27,8 @@
 // Common page class
 .page-content {
     .page-placeholder {
-        color: $page-icon-color;
-        font-size: $page-content-placeholder-font-size;
+        color: $grey;
```
Comment (bundyo) [inline packages/template-tab-navigation-ng/src/_app-common.scss]: Better use const() to get these, as free floating variables would be used just for input.

File: packages/template-tab-navigation-ng/package.json [hunk 3]
```diff
@@ -50,10 +50,10 @@
     "@angular/platform-browser-dynamic": "~8.2.0",
     "@angular/router": "~8.2.0",
     "nativescript-angular": "~8.2.0",
-    "nativescript-theme-core": "~1.0.6",
+    "nativescript-theme-core": "~2.0.11",
     "reflect-metadata": "~0.1.12",
     "rxjs": "^6.4.0",
-    "tns-core-modules": "next",
+    "tns-core-modules": "^6.1.0-rc-2019-08-30-092856-01",
```
Comment (bundyo) [inline packages/template-tab-navigation-ng/package.json]: You can set 6.1.0 now that the rc is up :)

File: packages/template-tab-navigation-ng/src/app.android.scss (hunk 1 above)
Comment (MartoYankov) [reply packages/template-tab-navigation-ng/src/app.android.scss]: This is something that was decided about the templates some time ago - we wanted to show at least in one place how to have a common file + branching platform specific files. Basically, boilerplate.

**Comment by manoldonev** [comment]
test

**Pull request merged**